federation/pkg/kubefed/init/init.go

```diff
@@ -638,20 +640,23 @@ func createControllerManagerKubeconfigSecret(clientset client.Interface, namespa
 	return util.CreateKubeconfigSecret(clientset, config, namespace, kubeconfigName, name, "", dryRun)
 }
 
-func createPVC(clientset client.Interface, namespace, svcName, federationName, etcdPVCapacity string, dryRun bool) (*api.PersistentVolumeClaim, error) {
+func createPVC(clientset client.Interface, namespace, svcName, federationName, etcdPVCapacity, etcdPVStorageClass string, dryRun bool) (*api.PersistentVolumeClaim, error) {
 	capacity, err := resource.ParseQuantity(etcdPVCapacity)
 	if err != nil {
 		return nil, err
 	}
 
+	annotations := map[string]string{federation.FederationNameAnnotation: federationName}
+	if len(etcdPVStorageClass) > 0 {
+		annotations["volume.beta.kubernetes.io/storage-class"] = etcdPVStorageClass
```

Comment:
This is very different than the version skew conversation we are having right now. The current conversation is about the skew between the cluster versions and the federation control plane version. This conversation is about hosting the federation control plane. The answers to these two could be very different. 

It seems odd to me that we are now going to say that users can't host federation control plane in clusters older than 1.6 using kubefed while they could with kubefed 1.5 and 1.6. I don't feel too strongly about it, but in the absence of a well defined version skew statement, I would rather be liberal and use the beta annotation to accommodate 1.5 clusters. We don't have to worry about clusters older than 1.5 since kubefed did not even exist before then.